### ISSUE TITLE
✅ 🐞 Bug Report: Can't update to Elementor Pro 3.11.2 - [Ed-10136]

### ISSUE THREAD

**Issue by cestrian77** [elementor/elementor#21460]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

When attempting to update a range of different sites to Elementor 3.11.2 (from 3.11.1) I get an "Update failed: -1" error message. When looking in the Elementor >> System Info  >> Log there are a range of PHP issues being reported - see below.

````
PHP: 2023-02-23 11:11:03 [notice X 67][/var/www/vhosts/DOMAIN/httpdocs/wp-content/plugins/elementor-pro/modules/loop-builder/module.php::200] strpos(): Passing null to parameter #1 ($haystack) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-02-23 11:11:55 [notice X 2][/var/www/vhosts/DOMAIN/httpdocs/wp-content/plugins/elementor-pro/core/database/model-base.php::135] Return type of ElementorPro\Core\Database\Model_Base::jsonSerialize() should either be compatible with JsonSerializable::jsonSerialize(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-02-23 11:12:38 [notice X 6][/var/www/vhosts/DOMAIN/httpdocs/wp-content/plugins/elementor-pro/core/utils/collection.php::75] Return type of ElementorPro\Core\Utils\Collection::jsonSerialize() should either be compatible with JsonSerializable::jsonSerialize(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-02-23 11:12:45 [notice X 5][/var/www/vhosts/DOMAIN/httpdocs/wp-content/plugins/elementor-pro/core/database/model-base.php::135] Return type of ElementorPro\Core\Database\Model_Base::jsonSerialize() should either be compatible with JsonSerializable::jsonSerialize(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice [array (
  'trace' => '
#0: /var/www/vhosts/DOMAIN/httpdocs/wp-content/plugins/elementor-pro/core/database/model-base.php(10): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /var/www/vhosts/DOMAIN/httpdocs/wp-content/plugins/elementor-pro/plugin.php(159): class type include()array (
  0 => '/var/www/vhosts/DOMAIN/httpdocs/wp-content/plugins/elementor-pro/core/database/model-base.php',
)
#2: /var/www/vhosts/DOMAIN/httpdocs/wp-content/plugins/elementor-pro/modules/notes/database/models/note.php(18): ElementorPro\Plugin -> autoload()
#3: /var/www/vhosts/DOMAIN/httpdocs/wp-content/plugins/elementor-pro/plugin.php(159): class type include()array (
  0 => '/var/www/vhosts/DOMAIN/httpdocs/wp-content/plugins/elementor-pro/modules/notes/database/models/note.php',
)
#4: /var/www/vhosts/DOMAIN/httpdocs/wp-content/plugins/elementor-pro/modules/notes/data/controller.php(243): ElementorPro\Plugin -> autoload()
',
)]
````

### Steps to reproduce

1. Go to Plugins >> Installed Plugins
2. Click on 'update now' next to Elementor Pro v 3.11.1 to update to version 3.11.2

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt


== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MariaDB Server v5.5.61-MariaDB-38.13
	PHP Version: 8.1.16
	PHP Memory Limit: 128M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 16M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.1.1
	Site URL: https://wizardly-hellman.92-205-26-83.plesk.page
	Home URL: https://wizardly-hellman.92-205-26-83.plesk.page
	WP Multisite: No
	Max Upload Size: 16 MB
	Memory limit: 40M
	Max Memory limit: 256M
	Permalink Structure: /index.php/%year%/%monthnum%/%day%/%postname%/
	Language: en-GB
	Timezone: 0
	Admin Email: matt.cody@bluestonesgroup.co.uk
	Debug Mode: Inactive

== Theme ==
	Name: APEX theme v3
	Version: 3.0.0
	Author: Matt Cody
	Child Theme: Yes
	Parent Theme Name: Astra
	Parent Theme Version: 4.0.2
	Parent Theme Author: Brainstorm Force

== User ==
	Role: administrator
	WP Profile lang: en_GB
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36

== Active Plugins ==
	Classic Editor
		Version: 1.6.2
		Author: WordPress Contributors

	Cookie Notice &amp; Compliance for GDPR / CCPA
		Version: 2.4.6
		Author: Hu-manity.co

	Duplicate Page
		Version: 4.5.1
		Author: mndpsingh287

	Elementor
		Version: 3.11.2
		Author: Elementor.com

	Elementor Pro
		Version: 3.11.1
		Author: Elementor.com

	Email Log
		Version: 2.4.8
		Author: Sudar

	Gravity Forms
		Version: 2.7.1
		Author: Gravity Forms

	Gravity PDF
		Version: 6.5.4
		Author: Blue Liquid Designs

	ManageWP - Worker
		Version: 4.9.16
		Author: GoDaddy

	Premium Starter Templates
		Version: 3.1.25
		Author: Brainstorm Force

	Ultimate Addons for Elementor
		Version: 1.36.14
		Author: Brainstorm Force

	WP All Import Pro
		Version: 4.7.8
		Author: Soflyy

	WP Fastest Cache
		Version: 1.1.1
		Author: Emre Vona

	WP Mail SMTP
		Version: 3.7.0
		Author: WPForms

	Yoast Duplicate Post
		Version: 4.5
		Author: Enrico Battocchi & Team Yoast

	Yoast SEO
		Version: 20.1
		Author: Team Yoast


== Must-Use Plugins ==
	ManageWP - Worker Loader
		Version: 1.0.0
		Author: GoDaddy


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	Optimised DOM Output: Active by default
	Improved asset loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive by default
	Accessibility improvements: Active by default
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Inactive by default
	Upgrade Swiper Library: Inactive by default
	Default to New Theme Builder: Active by default
	Editor V2: Inactive by default
	Landing pages: Active by default
	Nested Elements: Inactive by default
	Lazy Load Background Images: Inactive by default
	Save as Default: Active by default
	Page Transitions: Active by default
	Notes: Active by default
	Loop: Active by default
	Form Submissions: Active by default
	Scroll Snap: Active by default
	Mega Menu: Inactive by default


== Log ==
	
PHP: showing 4 of 4PHP: 2023-02-23 11:11:03 [notice X 71][/var/www/vhosts/wizardly-hellman.92-205-26-83.plesk.page/httpdocs/wp-content/plugins/elementor-pro/modules/loop-builder/module.php::200] strpos(): Passing null to parameter #1 ($haystack) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-02-23 11:11:55 [notice X 2][/var/www/vhosts/wizardly-hellman.92-205-26-83.plesk.page/httpdocs/wp-content/plugins/elementor-pro/core/database/model-base.php::135] Return type of ElementorPro\Core\Database\Model_Base::jsonSerialize() should either be compatible with JsonSerializable::jsonSerialize(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-02-23 11:12:38 [notice X 6][/var/www/vhosts/wizardly-hellman.92-205-26-83.plesk.page/httpdocs/wp-content/plugins/elementor-pro/core/utils/collection.php::75] Return type of ElementorPro\Core\Utils\Collection::jsonSerialize() should either be compatible with JsonSerializable::jsonSerialize(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-02-23 11:12:45 [notice X 5][/var/www/vhosts/wizardly-hellman.92-205-26-83.plesk.page/httpdocs/wp-content/plugins/elementor-pro/core/database/model-base.php::135] Return type of ElementorPro\Core\Database\Model_Base::jsonSerialize() should either be compatible with JsonSerializable::jsonSerialize(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice [array (
  'trace' => '
#0: /var/www/vhosts/wizardly-hellman.92-205-26-83.plesk.page/httpdocs/wp-content/plugins/elementor-pro/core/database/model-base.php(10): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /var/www/vhosts/wizardly-hellman.92-205-26-83.plesk.page/httpdocs/wp-content/plugins/elementor-pro/plugin.php(159): class type include()array (
  0 => '/var/www/vhosts/wizardly-hellman.92-205-26-83.plesk.page/httpdocs/wp-content/plugins/elementor-pro/core/database/model-base.php',
)
#2: /var/www/vhosts/wizardly-hellman.92-205-26-83.plesk.page/httpdocs/wp-content/plugins/elementor-pro/modules/notes/database/models/note.php(18): ElementorPro\Plugin -> autoload()
#3: /var/www/vhosts/wizardly-hellman.92-205-26-83.plesk.page/httpdocs/wp-content/plugins/elementor-pro/plugin.php(159): class type include()array (
  0 => '/var/www/vhosts/wizardly-hellman.92-205-26-83.plesk.page/httpdocs/wp-content/plugins/elementor-pro/modules/notes/database/models/note.php',
)
#4: /var/www/vhosts/wizardly-hellman.92-205-26-83.plesk.page/httpdocs/wp-content/plugins/elementor-pro/modules/notes/data/controller.php(243): ElementorPro\Plugin -> autoload()
',
)]

Log: showing 11 of 112023-02-23 11:13:19 [info] elementor-pro::elementor_pro_updater Started 
2023-02-23 11:13:19 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-02-23 11:13:19 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-02-23 11:13:19 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.1',
  'to' => '3.11.2',
)]
2023-02-23 11:13:19 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.1',
  'to' => '3.11.2',
)]
2023-02-23 11:13:21 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.1',
  'to' => '3.11.2',
)]
2023-02-23 11:16:32 [info] elementor::elementor_updater Started 
2023-02-23 11:16:32 [info] Elementor/Upgrades - _on_each_version Start  
2023-02-23 11:16:32 [info] Elementor/Upgrades - _on_each_version Finished 
2023-02-23 11:16:32 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.1',
  'to' => '3.11.2',
)]
2023-02-23 11:16:32 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.1',
  'to' => '3.11.2',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatibility not specified
	Ultimate Addons for Elementor: Compatibility not specified

== Elementor Pro - Compatibility Tag ==
	

````
</details>

**Comment by PocketTim** [comment]
Exact same issue here. Ignoring the WP Updater and trying to manually install 3.11.2 from file also produces the same -1 error.

**Comment by cestrian77** [comment]
Yeah - I tried to do the manual update route too. Same result. Had to delete the maintenance file on the server just to get the site out of maintenance mode.

**Comment by PocketTim** [comment]
Might be worth adding that Elementor Pro and Starter Templates are also affected by this bug. 

All of my other plugins appear to be updating without issue.

**Comment by berkmh** [comment]
Just curious, is this happening only with sites on PHP 8.1 or is it affecting 8.0 and 8.2 also?

**Comment by PocketTim** [comment]
> Just curious, is this happening only with sites on PHP 8.1 or is it affecting 8.0 and 8.2 also?

All of my sites are on 7.4.

**Comment by nicholaszein** [comment]
Hello everyone! 

Thank you for submitting your issue, and thank you for the feedback! 🙏 

If you can, please share which hosting services you are using. For those who commented, it would also be great if you could share your Elementor > System Info as well, so we can see if we can find a common denominator. This would be very helpful, and we would very much appreciate it.

Please allow us some time to investigate it, and get back to you with our findings. 🧐 

Best regards

**Comment by PocketTim** [comment]
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: mariadb.org binary distribution v10.5.17
	PHP Version: 8.1.16
	PHP Memory Limit: 1024M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 128M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.1.1
	Site URL: https://retrogamingmags.com
	Home URL: https://retrogamingmags.com
	WP Multisite: No
	Max Upload Size: 128 MB
	Memory limit: 40M
	Max Memory limit: 1024M
	Permalink Structure: /%postname%
	Language: en-GB
	Timezone: 0
	Admin Email: REDACTED
	Debug Mode: Inactive

== Theme ==
	Name: Astra
	Version: 4.0.2
	Author: Brainstorm Force
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_GB
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/16.3 Safari/605.1.15

== Active Plugins ==
	Add From Server
		Version: 3.4.5
		Author: Dion Hulse

	Better Search Replace
		Version: 1.4.2
		Author: WP Engine

	Cloudflare
		Version: 4.11.0
		Author: Cloudflare, Inc.

	dflip
		Version: 1.7.35
		Author: DearHive

	Elementor
		Version: 3.11.1
		Author: Elementor.com

	Elementor Pro
		Version: 3.11.1
		Author: Elementor.com

	GA Google Analytics
		Version: 20221016
		Author: Jeff Starr

	MailChimp
		Version: 1.5.8
		Author: MailChimp

	OneSignal Push Notifications
		Version: 2.3.3
		Author: OneSignal

	Simple Banner
		Version: 2.13.3
		Author: Ryan Petersen

	Starter Templates
		Version: 3.1.25
		Author: Brainstorm Force

	UpdraftPlus - Backup/Restore
		Version: 1.22.24
		Author: UpdraftPlus.Com, DavidAnderson

	WP Duplicate Page
		Version: 1.3
		Author: Ninja Team

	WP Fastest Cache
		Version: 1.1.1
		Author: Emre Vona

	Yoast SEO
		Version: 20.1
		Author: Team Yoast


== Must-Use Plugins ==
	1&1 Product Subdomain
		Version: 1.1.0
		Author: 1&1


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	Optimised DOM Output: Active by default
	Improved asset loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive by default
	Accessibility improvements: Active by default
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Inactive by default
	Upgrade Swiper Library: Inactive by default
	Default to New Theme Builder: Active by default
	Editor V2: Inactive by default
	Landing pages: Active by default
	Nested Elements: Inactive by default
	Lazy Load Background Images: Inactive by default
	Save as Default: Active by default
	Page Transitions: Active by default
	Notes: Active by default
	Loop: Active by default
	Form Submissions: Active by default
	Scroll Snap: Active by default
	Mega Menu: Inactive by default


== Log ==
	
Log: showing 20 of 332023-01-30 06:27:45 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.10.2',
  'to' => '3.10.3',
)]
2023-02-14 06:26:41 [info] elementor-pro::elementor_pro_updater Started 
2023-02-14 06:26:41 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-02-14 06:26:41 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-02-14 06:26:41 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.10.3',
  'to' => '3.11.0',
)]
2023-02-14 06:26:41 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.10.3',
  'to' => '3.11.0',
)]
2023-02-14 16:56:16 [info] elementor::elementor_updater Started 
2023-02-14 16:56:16 [info] Elementor/Upgrades - _on_each_version Start  
2023-02-14 16:56:16 [info] Elementor/Upgrades - _on_each_version Finished 
2023-02-14 16:56:16 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.10.2',
  'to' => '3.11.0',
)]
2023-02-16 06:27:17 [info] elementor::elementor_updater Started 
2023-02-16 06:27:17 [info] Elementor/Upgrades - _on_each_version Start  
2023-02-16 06:27:17 [info] Elementor/Upgrades - _on_each_version Finished 
2023-02-16 06:27:17 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.0',
  'to' => '3.11.1',
)]
2023-02-16 06:27:17 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.0',
  'to' => '3.11.1',
)]
2023-02-16 10:58:15 [info] elementor-pro::elementor_pro_updater Started 
2023-02-16 10:58:15 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-02-16 10:58:15 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-02-16 10:58:15 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.0',
  'to' => '3.11.1',
)]
2023-02-16 10:58:15 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.0',
  'to' => '3.11.1',
)]

JS: showing 3 of 3JS: 2023-01-19 19:33:41 [error X 1][https://retrogamingmags.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.10.1:3:296806] undefined is not an object (evaluating 'T.params.args.data.id') 
JS: 2023-01-27 13:57:47 [error X 7][https://retrogamingmags.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.10.1:3:782020] undefined is not an object (evaluating 'this.store.find((function(T){return T.original===C})).localized') 
JS: 2023-01-27 13:57:50 [error X 1][https://retrogamingmags.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.10.1:3:781512] undefined is not an object (evaluating 'T.originalEvent.key.toLowerCase') 

PHP: showing 6 of 6PHP: 2023-01-19 20:12:42 [notice X 1][/homepages/2/d638404950/htdocs/clickandbuilds/SegaMagazines/wp-content/plugins/elementor/includes/managers/image.php::108] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-02-03 12:51:16 [notice X 1][/homepages/2/d638404950/htdocs/clickandbuilds/SegaMagazines/wp-content/plugins/elementor/core/common/modules/ajax/module.php::172] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-02-16 11:09:47 [error X 1][/homepages/2/d638404950/htdocs/clickandbuilds/SegaMagazines/wp-content/plugins/elementor/includes/template-library/sources/local.php::1080] Uncaught TypeError: Argument 2 passed to Elementor\TemplateLibrary\Source_Local::on_save_post() must be an instance of WP_Post, null given, called in /homepages/2/d638404950/htdocs/clickandbuilds/SegaMagazines/wp-includes/class-wp-hook.php on line 310 and defined in /homepages/2/d638404950/htdocs/clickandbuilds/SegaMagazines/wp-content/plugins/elementor/includes/template-library/sources/local.php:1080
Stack trace:
#0 /homepages/2/d638404950/htdocs/clickandbuilds/SegaMagazines/wp-includes/class-wp-hook.php(310): Elementor\TemplateLibrary\Source_Local->on_save_post(6685, NULL)
#1 /homepages/2/d638404950/htdocs/clickandbuilds/SegaMagazines/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters(NULL, Array)
#2 /homepages/2/d638404950/htdocs/clickandbuilds/SegaMagazines/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#3 /homepages/2/d638404950/htdocs/clickandbuilds/SegaMagazines/wp-includes/post.php(4717): do_action('save_post', 6685, NULL, true)
#4 /homepages/2/d638404950/htdocs/clickandbuilds/SegaMaga [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-02-24 15:24:11 [notice X 7][/homepages/2/d638404950/htdocs/clickandbuilds/SegaMagazines/wp-content/plugins/elementor-pro/core/utils/collection.php::75] Return type of ElementorPro\Core\Utils\Collection::jsonSerialize() should either be compatible with JsonSerializable::jsonSerialize(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-02-24 15:24:11 [notice X 766][/homepages/2/d638404950/htdocs/clickandbuilds/SegaMagazines/wp-content/plugins/elementor-pro/modules/loop-builder/module.php::200] strpos(): Passing null to parameter #1 ($haystack) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-02-24 17:14:49 [notice X 6][/homepages/2/d638404950/htdocs/clickandbuilds/SegaMagazines/wp-content/plugins/elementor-pro/core/database/model-base.php::135] Return type of ElementorPro\Core\Database\Model_Base::jsonSerialize() should either be compatible with JsonSerializable::jsonSerialize(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice [array (
  'trace' => '
#0: /homepages/2/d638404950/htdocs/clickandbuilds/SegaMagazines/wp-content/plugins/elementor-pro/core/database/model-base.php(10): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: /homepages/2/d638404950/htdocs/clickandbuilds/SegaMagazines/wp-content/plugins/elementor-pro/plugin.php(159): class type include()array (
  0 => '/homepages/2/d638404950/htdocs/clickandbuilds/SegaMagazines/wp-content/plugins/elementor-pro/core/database/model-base.php',
)
#2: /homepages/2/d638404950/htdocs/clickandbuilds/SegaMagazines/wp-content/plugins/elementor-pro/modules/notes/database/models/note.php(18): ElementorPro\Plugin -> autoload()
#3: /homepages/2/d638404950/htdocs/clickandbuilds/SegaMagazines/wp-content/plugins/elementor-pro/plugin.php(159): class type include()array (
  0 => '/homepages/2/d638404950/htdocs/clickandbuilds/SegaMagazines/wp-content/plugins/elementor-pro/modules/notes/database/models/note.php',
)
#4: /homepages/2/d638404950/htdocs/clickandbuilds/SegaMagazines/wp-content/plugins/elementor-pro/modules/notes/data/controller.php(243): ElementorPro\Plugin -> autoload()
',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatibility not specified

== Elementor Pro - Compatibility Tag ==

**Comment by cestrian77** [comment]
We're using a self-managed VPS via Heart Internet for hosting.

**Comment by japierce1028** [comment]
I am using Dreamhost PHP 8.1. Multiple sites have this issue.

**Comment by PocketTim** [comment]
I found a workaround for this issue but it only works if you also have the Starter Templates plugin installed.

Disable the Starter Templates plugin. Update Starter Templates. Then update everything else and then reenable Starter Templates.

**Comment by Fokang-Juan** [comment]
There is a bug with Starter Template Plugin.
i found the solution here: [](https://www.reddit.com/r/Wordpress/comments/119zk8v/update_failed/)
It work for me: 
To update to the fixed version of the plugin:

-     Deactivate the Starter Templates plugin (since it’s breaking the updater)
-    Update the Starter Templates plugin to 3.1.26 or higher
-     Reactivate the Starter Templates plugin

**Comment by Nevoss** [comment]
It seems like these issues cause by PHP 8.1 please try to use 7.4 or 8.0

**Comment by PocketTim** [comment]
> It seems like these issues cause by PHP 8.1 please try to use 7.4 or 8.0

I proved above that this is not the case, running any version of PHP makes no difference.

Disabling Starter Templates and then updating as two of us have mentioned above is the only fix for this at the moment.

**Comment by Spider159** [comment]
> Exact same issue here. Ignoring the WP Updater and trying to manually install 3.11.2 from file also produces the same -1 error.

i am facing the same issue i try every single thing but its not working

**Comment by Spider159** [comment]
> found a workaround for this issue but it only works if you also have the Starter Templates plugin installed.

i am using { hello theme  } and i am facing the same issue ,,,

**Comment by Spider159** [comment]
> Hello everyone!
> 
> Thank you for submitting your issue, and thank you for the feedback! 🙏
> 
> If you can, please share which hosting services you are using. For those who commented, it would also be great if you could share your Elementor > System Info as well, so we can see if we can find a common denominator. This would be very helpful, and we would very much appreciate it.
> 
> Please allow us some time to investigate it, and get back to you with our findings. 🧐
> 
> Best regards

**Comment by Spider159** [comment]
> Hello everyone!
> 
> Thank you for submitting your issue, and thank you for the feedback! 🙏
> 
> If you can, please share which hosting services you are using. For those who commented, it would also be great if you could share your Elementor > System Info as well, so we can see if we can find a common denominator. This would be very helpful, and we would very much appreciate it.
> 
> Please allow us some time to investigate it, and get back to you with our findings. 🧐
> 
> Best regards

please fix the issue as soon as possible its a bug & its nothing to do with our end

**Comment by nicholaszein** [comment]
Hello everyone,

we are investigating this issue. Can everyone confirm that one of the common denominators is the use of BrainstormForce's Starter Templates plugin (free or premium)?

**Comment by nicholaszein** [comment]
Hey everyone!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Pro **v3.12.3**! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

Cheers 🥂
